### PR TITLE
Update rcdevice.c

### PR DESCRIPTION
**Pull request by FLahner23**
See issue:
https://github.com/iNavFlight/inav/issues/9343

### CODE REVIEW

File: src/main/io/rcdevice.c [hunk 1]
```diff
@@ -270,6 +270,10 @@ static void runcamDeviceParseV2DeviceInfo(rcdeviceResponseParsingContext_t *ctx)
 // definition of runcamDeviceInfo_t to know more)
 static void runcamDeviceGetDeviceInfo(runcamDevice_t *device)
 {
+    // preset with rc split 2s support which has issue to respond
```
Comment (mmosca) [inline src/main/io/rcdevice.c:273]: This probably should not be blindly enabled, as we are asking the device for information that should populate the features.

We should probably fix the protocol differences with this camera, so we can read the featrures correctly, or at least have a configuration variable to enable this hack.

Comment (FLahner23) [reply src/main/io/rcdevice.c:273]: Change it as you like it is hot fix for now. Having feature variable would be good for diagnostics via CLI.Also I noticed that runcam attempt interval variable was removed in some Inav version as there was related workaround before.
I would add possibility to set features manually and let user to configure/map what he needs.

**Comment by b14ckyy** [comment]
Are you still planning to provide this? When validated, we can merge it for 8.0

**Comment by FLahner23** [comment]
> Are you still planning to provide this? When validated, we can merge it for 8.0

Dont know how to add new parameter to iNav e.g. RuncamSplitEnable. When enabled the added lines of code will be enabled. If you know that you can you add this parameter handling here and move it to validation

**Pull request closed without merging**